apps/explorer/src/components/HomeMetrics/Checkpoint.tsx

```diff
@@ -2,15 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { StatsWrapper } from './FormattedStatsAmount';
-import { NetworkStats } from './NetworkStats';
 
 import { useGetNetworkMetrics } from '~/hooks/useGetNetworkMetrics';
+import { Card } from '~/ui/Card';
 
 export function Checkpoint() {
     const { data, isLoading } = useGetNetworkMetrics();
 
     return (
-        <NetworkStats spacing="none">
+        <Card height="full" spacing="lg">
```

Comment:
I switched these components to just render a `Card` because `NetworkStats` had styling for the specific `OnTheNetwork` component which was making it hard to style these components properly